packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -20,6 +20,11 @@ export type TooltipProps = React.PropsWithChildren<
     text: string
     type?: 'label' | 'description'
     keybindingHint?: KeybindingHintProps['keys']
+    /**
+     * Delay in milliseconds before showing the tooltip
+     * @default 50
+     */
+    delay?: number
```

Comment:
I really think this is a risky thing to do.

Here's my preference (in order), we should get primer designers to weigh in
1) If we can, have the delay baked into the component instead of instance
2) If we do have to add a delay on an instance, it's one fixed number, not something you have to decide. (`delayed: boolean` instead of `delay: number`)

Reply:
Yeah totally fair. I am not comfortable to have this as a free text field either, just wanted to push something as we are not super clear about the direction 😄